.github/workflows/build-size-comparison.yml

```
@@ -0,0 +1,353 @@
+        
+        # Move zip files to root directory for easier access
+        cd ../../../
+        ls -la *.zip
+
+    - name: Extract and Compare Files
+      run: |
+        # Extract both zip files
+        mkdir -p current-files base-files
+        unzip -q current-build.zip -d current-files/
+        unzip -q base-build.zip -d base-files/
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)